opentech/apply/activity/messaging.py

```diff
@@ -128,7 +128,7 @@ def process_batch(self, message_type, events, request, user, submissions, relate
             self.process_send(message_type, recipients, events, request, user, submissions=submissions, submission=None, related=related, **kwargs)
 
     def process(self, message_type, event, request, user, submission, related=None, **kwargs):
-        recipients = self.recipients(message_type, submission=submission, **kwargs)
+        recipients = self.recipients(message_type, submission=submission, related=related, **kwargs)
```

Comment:
Added the "related" argument. Will other instances of "recipients" functions complain if it's not present?

Reply:
There will be no issue as other `recipients` functions will have `related` available in `kwargs` if not explicitly defined.